python/.pre-commit-config.yaml

```diff
@@ -39,7 +39,7 @@ repos:
       - id: ruff-format
   - repo: https://github.com/astral-sh/uv-pre-commit
     # uv version.
-    rev: 0.5.30
+    rev: 0.9.8
```

Comment:
Is there no "latest" tag we can use so we don't have to continuously update?

Reply:
we could, but pre-commit doesn't like it and neither does git...